content/post/proficiency_v_creativity.md

```diff
@@ -0,0 +1,69 @@
+
+Creating a team culture that values these behaviors (ensuring they don’t become under-appreciated [glue work](https://www.locallyoptimistic.com/post/glue-work/)) increases the reliability of results and protects the institutional capital of the Analytics team. We’ve covered a lot of tactics here on Locally Optimistic to build the [data quality flywheel](https://www.locallyoptimistic.com/post/data-dies-in-darkness/) and improve the focus on proficiency within the data team (with help from the rest of the company).
+
+One other tactic I’ve found incredibly useful is to define user trust as a key metric for the data team. It’s one thing to know that occasionally, things break - it isn’t ideal but you’re working to improve it. It’s another thing to know that because something occasionally breaks, your colleagues have trouble trusting the output of the data team more generally. To get that kind of action-inspiring feedback, we survey the company about every 4 months. Questions cover whether folks can access the data they need, understand the data they have, and most importantly to me, whether they trust the data resources they use. Keeping that number moving in the right direction helps motivate the team to ensure proficiency. Fair warning, though - the results from the first survey can be very humbling, so make sure you’re ready to embrace the feedback as constructive!
+
+But as much as I value a robust data warehouse and [accurate reporting](https://www.locallyoptimistic.com/post/reporting-is-a-gateway-drug/), that isn’t what has kept me working with data over the years.
+
+## Creativity
+The creativity of analytics is why I show up to work each morning, and I think that’s true for many of us. We want to turn data into new knowledge and products - to learn from the past in order to shape the future. We want to explore and uncover new insights that suggest a path forward for the business. We want to predict who is going to churn instead of reporting on who churned last week. We want to create beautiful and signal-rich visualizations that allow our colleagues to immediately grasp relevant trends and patterns. And though I argued that the *outputs* of analytics engineering encourage proficiency just a few paragraphs ago, I think the *emergence* of analytics engineering as a discipline is also an act of creativity. Very smart folks are thinking outside the box to create tools and processes that apply engineering best practices and solutions to the problems analysts face.
+
+Leading for creativity means fostering systems that consistently churn out fresh ideas, even if many of those ideas don’t turn out to be successful. Your team has to understand that failure is not just a possibility, it's a necessary output of both research and creative processes. Every failure is an opportunity to change your perspective, reframe either the problem or the solution, and over time hone in on a more refined idea of what approaches are likely to be fruitful.
```

Comment:
I *think* you're looking to `home in on a more refined idea`. Unless you're `honing your refined idea`.